cmd/fil-daggregator/README.md

```diff
@@ -9,15 +9,26 @@ fil-daggregator
 go install github.com/filecoin-project/go-dagaggregator-unixfs/cmd/fil-daggregator@latest
 ```
 
-## Usage Example
+## Usage Example #1
+Creating a DAG-aggregate and exporting it to a .car file. Assumes [go-ipfs](https://github.com/ipfs/go-ipfs) is installed.
+
+```
+ipfs daemon --offline
+ipfs add file1  # output: QmNpiuBaHgQJP5KatAN2sqoW5p2eUp35nYzPXieNQmmHja
+ipfs add file2  # output: QmNpiuBaHgQJP5KatAN2sqoW5p2eUp35nYzPXieNQmmHja
```

Comment:
@dchoi27 note that this will result in a single-entry aggregate ( the aggregator always de-duplicates ). Your example is totally valid and does work, I am just worrying it sets the wrong expectations.